src/router/router.ts

```diff
@@ -196,6 +208,31 @@ export class DanetHTTPRouter {
 		};
 	}
 
+	private handleSSE(executionContext: ExecutionContext, response: EventTarget) {
+		return streamSSE(executionContext, async (stream: SSEStreamingApi) => {
+			let canContinue = true;
+			(response as unknown as EventTarget).addEventListener(
```

Comment:
Isn't response an `EventTarget` already?

```suggestion
			response.addEventListener(
```

Reply:
Yes it is, but this is not a cast, it's only a TS compile time thing so it does not complain about it not being an EventTarget because the typing before is only `string |  Record` to not break existing code.

Reply:
Sorry, but I'm not sure if I understood 🤔 

I thought the issue you mentioned was solved on line `:191` while calling `handleSSE`.
Inside this function, the `response` parameter was defined as `EventTarget`, so I though the `as unknown as EventTarget` was not needed.

Reply:
You are 100% right. I forgot that I moved that code to handleSSE where typing is properly done. My bad !